docker/Dockerfile.deps

```diff
@@ -160,7 +160,18 @@ RUN LIBSND_VERSION=1.0.28 && cd /tmp
     rm libsndfile-$LIBSND_VERSION.tar.gz                                                                       && \
     cd libsndfile-$LIBSND_VERSION                                                                              && \
     ./configure && make -j"$(grep ^processor /proc/cpuinfo | wc -l)" && make install                           && \
-    cd /tmp && rm -rf libsnd-$LIBSND_VERSION
+    cd /tmp && rm -rf libsndfile-$LIBSND_VERSION
 
 # CUDA
 COPY --from=cuda /usr/local/cuda /usr/local/cuda
+
+# add devtoolset-7 to paths, it won't break anyting even if devtoolset-7 are not installed
+ENV ADDITIONAL_DEVTOOLSET_ROOTPATH /opt/rh/devtoolset-7/root
+ENV PATH $ADDITIONAL_DEVTOOLSET_ROOTPATH/usr/bin:$PATH
+ENV LD_LIBRARY_PATH $ADDITIONAL_DEVTOOLSET_ROOTPATH/usr/lib64:$ADDITIONAL_DEVTOOLSET_ROOTPATH/usr/lib:$ADDITIONAL_DEVTOOLSET_ROOTPATH/usr/lib64/dyninst:$ADDITIONAL_DEVTOOLSET_ROOTPATH/usr/lib/dyninst:$LD_LIBRARY_PATH
```

Comment:
```suggestion
# add devtoolset-7 to paths, it won't break anyting even if devtoolset-7 are not installed
ENV ADDITIONAL_DEVTOOLSET_ROOTPATH=/opt/rh/devtoolset-7/root    \
    PATH=$ADDITIONAL_DEVTOOLSET_ROOTPATH/usr/bin:$PATH         \
    LD_LIBRARY_PATH=$ADDITIONAL_DEVTOOLSET_ROOTPATH/usr/lib64:$ADDITIONAL_DEVTOOLSET_ROOTPATH/usr/lib:$ADDITIONAL_DEVTOOLSET_ROOTPATH/usr/lib64/dyninst:$ADDITIONAL_DEVTOOLSET_ROOTPATH/usr/lib/dyninst:$LD_LIBRARY_PATH
```
Thanks to this we'll have only one additional layer in docker

Reply:
Done

Reply:
You missed the `=` char ;)
```
[...]ROOTPATH=/opt/rh[...]
             ^ this one
```
Linking ref to docs: https://docs.docker.com/engine/reference/builder/#env

Reply:
Done